cloudinit/distros/gentoo.py

```diff
@@ -220,7 +226,7 @@ def set_timezone(self, tz):
         distros.set_etc_timezone(tz=tz, tz_file=self._find_tz_file(tz))
 
     def package_command(self, command, args=None, pkgs=None):
-        cmd = list("emerge")
+        cmd = ["emerge"]
```

Comment:
Why this change? Just curious

Reply:
Otherwise, it was showing up in the logs with an error, because it was trying to run something to the effect of "e m e r g e" as the command, rather than "emerge". I also remember seeing a bug report on launchpad for it with someone else reporting the same problem.

Reply:
Ouch. Nice catch.